Service/CreateAvailabilityService.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace CommonGateway\HuwelijksplannerBundle\Service;
+
+use App\Service\ObjectEntityService;
+use DateInterval;
+use DatePeriod;
+use DateTime;
+use Doctrine\ORM\EntityManagerInterface;
+use Exception;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use function Symfony\Component\DependencyInjection\Loader\Configurator\param;
+
+/**
+ * This service holds al the logic for creating availability.
+ */
+class CreateAvailabilityService
+{
+    private EntityManagerInterface $entityManager;
+    private ObjectEntityService $objectEntityService;
+    private SymfonyStyle $io;
+    private array $data;
+    private array $configuration;
+
+    /**
+     * @param ObjectEntityService    $objectEntityService
+     * @param EntityManagerInterface $entityManager
+     */
+    public function __construct(
+        ObjectEntityService $objectEntityService,
+        EntityManagerInterface $entityManager
+    ) {
+        $this->objectEntityService = $objectEntityService;
```

Comment:
You never use this objectEntityService service in this file?
And you should not because this service is old and depricated